LiskHQ/lisk-explorer
# Optimise UI implementation for tablet - Closes #483

### What was the problem?
`The new implementation of UI (refer to #442) introduced some regressions in Tablet (s and m media queries) screens. this ticket aims fix those issues only for screen resolutions between 768px and 1199px.

### How to test it?
 - Make sure nothing is broken in the target screen resolutions.
 - Make sure this is merged after #442 and #482

### Review checklist
- The PR solves #483
- All new code follows best practices
